ext/debase_internals.c

```diff
@@ -331,6 +333,13 @@ process_line_event(VALUE trace_point, void *data)
   tp = TRACE_POINT;
   path = rb_tracearg_path(tp);
 
+  if(context->stack_size == context->init_stack_size && context->hit_user_code) {
```

Comment:
because we might accidentally miss that stackframe (e.g. if an exception occurred)